lib/OpenCloud/Database/Resource/Instance.php

```diff
@@ -39,6 +39,7 @@ class Instance extends NovaResource
     public $created;
     public $updated;
     public $flavor;
+    public $backupRef;
```

Comment:
The problem here is with population. Since the JSON is structured `"restorePoint": {"backupRef": "foo"}`, iterators for example won't know what to do with the value when a DB instance is instantiated in a collection. One thing we can do is extend the base [populate method](https://github.com/rackspace/php-opencloud/blob/working/lib/OpenCloud/Common/Base.php#L292) like so:

```php
public function populate($info, $setObjects = true)
{
   parent::populate($info, $setObjects);

   if (is_object($info)) { 
      $info = (array) $info;
   }

   if (isset($info['restorePoint']['backupRef'])) {
      $this->backupRef = $info['restorePoint']['backupRef'];
   }
}
```